src/Functions/FunctionsStringHashFixedString.cpp

```diff
@@ -455,5 +480,24 @@ REGISTER_FUNCTION(HashFixedStrings)
         .category{"Hash"}});
 #    endif
 }
+
+#   if defined USE_ETHASH && USE_ETHASH
```

Comment:
This duplication is strange.
Do not use `defined`, so the compiler will check if you included the config.